### PR TITLE
Disable context auto saving when token persistence check fails

### PR DESCRIPTION
**Pull request by msJinLei**
## Description

- Disable Context auto saving when token persistence check fails in Windows and OSX

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Accounts/Accounts/ChangeLog.md [hunk 1]
```diff
@@ -19,6 +19,7 @@
 -->
 
 ## Upcoming Release
+* Disabled Context auto saving when token cache persistence fails
```
Comment (erich-wang) [inline src/Accounts/Accounts/ChangeLog.md:22]: How about make it more precise by adding "on Windows and MacOS"

**Review by erich-wang — CHANGES_REQUESTED** [review]

File: src/Accounts/Accounts/Properties/Resources.resx [hunk 2]
```diff
@@ -531,4 +531,7 @@
   <data name="PreviewFunctionMessage" xml:space="preserve">
     <value>This function is in preview. It may not be available in the selected subscription.</value>
   </data>
+  <data name="TokenCachePersistenceCheckError" xml:space="preserve">
+    <value>Persistence check fails of unknow error</value>
```
Comment (erich-wang) [inline src/Accounts/Accounts/Properties/Resources.resx:535]:
```suggestion
<value>Persistence check fails due to unknow error</value>
```

File: src/Accounts/Accounts/ChangeLog.md (hunk 1 above)
Comment (msJinLei) [reply src/Accounts/Accounts/ChangeLog.md]: Done

File: src/Accounts/Accounts/Properties/Resources.resx (hunk 2 above)
Comment (msJinLei) [reply src/Accounts/Accounts/Properties/Resources.resx]: Done

**Review by erich-wang — APPROVED** [review]

**Pull request merged**